opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go

```diff
@@ -137,8 +139,8 @@ resource "opentelekomcloud_compute_instance_v2" "instance_1" {
   image_id          = data.opentelekomcloud_images_image_v2.latest_image.id
   security_groups   = ["default"]
   availability_zone = "%s"
-  flavor_id         = "physical.o2.medium"
-  flavor_name       = "physical.o2.medium"
+  flavor_id         = "physical.h2.medium"
+  flavor_name       = "physical.h2.medium"
   tags              = ["foo", "bar"]
```

Comment:
Why not:
```
tags = {
    foo  = "bar"
    john = "doe"
  }
```

Reply:
@anton-sidelnikov 
https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/d5b7cd5487c66237234041e1c7ad38eb3229ac77/opentelekomcloud/services/bms/resource_opentelekomcloud_compute_bms_tags_v2.go#L44-L45

https://github.com/opentelekomcloud/gophertelekomcloud/blob/791d6bc332a31150be5b7f0e7c3ef66c54034feb/openstack/bms/v2/tags/create.go#L10